### DISCUSSION TITLE
wallet not syncing

### DISCUSSION THREAD

**Discussion by noseboy95** [Chia-Network/chia-blockchain#17647] [Support]
its been hours but just jammed. everything looks fine from my sight. not sure what is the issue here. please help

**Comment by BrandtH22** [comment]
Hey @noseboy95 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

Generally the first things I will do is turn it all off and back on, shut down chia and restart the device then start chia and see if you start syncing. 

If that does not work then I would go through this process:

Wallet or Node Sync Issues:
- Is it connecting to peers?

If yes:
- Is it downloading data from peers?
- Are there any errors in the logs?

If no to any of the above, reinstall a clean instance:
1. stop chia
2. uninstall the current version
3. rename the `~/.chia` folder to `~/.chia_old` (chia directory structure can be found here https://docs.chia.net/installation/#directory-structure)
4. install the latest chia version from here (https://www.chia.net/downloads/)
5. start chia

If chia loads after this then:
- Verify all custom settings are migrated to the new config file

If errors in the logs, relay those errors to our support team in discord for assistance https://discord.gg/chia

**Reply by noseboy95** [reply]
I'm not very tech guy. I did stop restart. I did uninstall everything except blockchains data. But it's still stuck at  4742289. ![PXL_20240305_223540596.jpg](https://github.com/Chia-Network/chia-blockchain/assets/162268711/02620641-304c-4334-a80e-d290208a54f7)


I believe it I connected to peers and trying to sync but not sure why it's stuck. I do still have more space.

**Reply by noseboy95** [reply]
Where can I find other nodes details to connect?

**Reply by BrandtH22** [reply]
Hey @noseboy95 ,  as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

To confirm whether you are connected to peers you can click the full node icon in the top right corner, it will show the peer connections and also the data being downloaded from them (this is also where you will add more peers). 

Generally it is best to have peers added via the introducer so you can add the nearest introducer peer from this list https://docs.chia.net/troubleshooting/node-syncing/#speed-up-connecting-to-nodes

If you do want to manually add nodes (generally not recommended) you can find lists of them on the community run platform alltheblocks: https://alltheblocks.net/chia There is a nodes/peers button on the right center of the screen.

It might be worthwhile to check your log files at `~/.chia/mainnet/logs/debug.log` to see if there are any WARNINGS or ERRORS related to peers 🤔

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
